src/utils.js

```diff
@@ -197,11 +200,32 @@ export const PopperPlacements = [
   'bottom-start',
   'left-end',
   'left',
-  'left-start',
+  'left-start'
 ];
 
 export const canUseDOM = !!(
   typeof window !== 'undefined' &&
   window.document &&
   window.document.createElement
 );
+
+export function findDOMElements(el) {
+  return canUseDOM && window.document.querySelectorAll(el);
+}
+
+export const defaultToggleEvents = ['touchstart', 'click'];
+
+export function addMultipleEventListeners(els, handler, events) {
+  events.forEach((event) => {
```

Comment:
Can you ensure events is an array before using it, it would be nice to make this works for single events which may be applied to multiple targets.